frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -1174,9 +1174,10 @@ function ReaderHighlight:exportToDocument(page, item)
     if can_write == false and not self.warned_once then
         self.warned_once = true
         UIManager:show(InfoMessage:new{
-            text = _([[The highlights you'll be making on this document will be saved in the setting file but won't be written in the document itself, as the file is in a read-only location.
+            text = _([[
+The highlights you'll be making on this document will be saved in the setting file but won't be written in the document itself, as the file is in a read-only location.
```

Comment:
(bit more succinct)
```suggestion
Highlights in this document will be saved in the settings file, but they won't be written in the document itself because the file is in a read-only location.
```